src/ResourceManagement/SqlManagement/Microsoft.Azure.Management.Sql/Generated/Models/Database.cs

```diff
@@ -0,0 +1,305 @@
+        /// <param name="type">Resource type</param>
+        /// <param name="tags">Resource tags</param>
+        /// <param name="collation">The collation of the Azure SQL
+        /// Database.</param>
+        /// <param name="creationDate">The creation date of the Azure SQL
+        /// Database (ISO8601 format).</param>
+        /// <param name="containmentState">The containment state of the Azure
+        /// SQL Database.</param>
+        /// <param name="currentServiceObjectiveId">The current Service Level
+        /// Objective Id of the Azure SQL Database. This is the Id of the
+        /// Service Level Objective that is currently active.</param>
+        /// <param name="databaseId">The Id of the Azure SQL Database.</param>
+        /// <param name="earliestRestoreDate">The recovery period start date
+        /// of the Azure SQL Database. This records the start date and time
+        /// when recovery is available for this Azure SQL Database (ISO8601
+        /// format).</param>
+        /// <param name="createMode">Specifies the type of database to create.
+        /// Possible values include: 'Copy', 'Default',
+        /// 'NonReadableSecondary', 'OnlineSecondary', 'PointInTimeRestore',
+        /// 'Recovery', 'Restore'</param>
+        /// <param name="sourceDatabaseId">Conditional.  Specifies the
+        /// resource Id of the source database.  If createMode is not set to
+        /// Default, then this value must be specified.</param>
+        /// <param name="edition">The edition of the Azure SQL Database.  The
+        /// DatabaseEditions enumeration contains all the valid editions.
+        /// Possible values include: 'Web', 'Business', 'Basic', 'Standard',
+        /// 'Premium', 'Free', 'Stretch', 'DataWarehouse'</param>
+        /// <param name="maxSizeBytes">The max size of the Azure SQL Database
+        /// expressed in bytes. Note: Only the following sizes are supported
+        /// (in addition to limitations being placed on each edition): { 100
+        /// MB | 500 MB |1 GB | 5 GB | 10 GB | 20 GB | 30 GB … 150 GB | 200
+        /// GB … 500 GB }</param>
+        /// <param name="requestedServiceObjectiveId">The configured Service
+        /// Level Objective Id of the Azure SQL Database. This is the Service
+        /// Level Objective that is in the process of being applied to the
+        /// Azure SQL Database.  Once successfully updated, it will match the
+        /// value of currentServiceObjectiveId property.</param>
+        /// <param name="requestedServiceObjectiveName">The name of the
+        /// configured Service Level Objective of the Azure SQL Database.
+        /// This is the Service Level Objective that is in the process of
+        /// being applied to the Azure SQL Database.  Once successfully
+        /// updated, it will match the value of serviceLevelObjective
+        /// property. Possible values include: 'Basic', 'S0', 'S1', 'S2',
+        /// 'S3', 'P1', 'P2', 'P3'</param>
```

Comment:
This is generated from the swagger, so I will address this in a separate PR for the specs.